<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDialog</class>
 <widget class="QDialog" name="EntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsername">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUrl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUsername">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUrl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGeneratePass">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
